platform/viewer/src/lib/DICOMSR/srUtils.js

```diff
@@ -0,0 +1,71 @@
+import OHIF from '@ohif/core';
```

Comment:
Drop the `utils` name/file, and make each of these methods their own file w/ default export?

Reply:
Just did this refactor, way cleaned and all with JSDocs 👍